utils/streams/transfers.mjs

```diff
@@ -0,0 +1,25 @@
+import map from 'through2-map';
+import * as models from '../../models';
+
+export const strTransfer = (inStream, outStream, callback) => {
+  inStream.pipe(map(callback)).pipe(outStream);
+};
+
+export const simpleTransfer = (inStream, outStream) => {
+  inStream.pipe(outStream);
+};
+
+export const csvToJsonTransfer = (inStream, outStream) => {

```

Comment:
There might be a problem:
Read data is stored to _data_ variable (e.g. RAM).
Parsed data is stored to _result _ variable (e.g. RAM).
This will lead us to a significant memory consumption.

The best approach is to read a file with a chunk per time, parse the read content and flush it. Then repeat those actions step by step until file is read totally. **through2.obj** method can save enough time.